CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Introduced `Container::get_host_port_ipv4` `Container::get_host_port_ipv6`,
+  `ContainerState::host_port_ipv4` and `ContainerState::host_port_ipv6` to better handle
```

Comment:
`ContainerState` is not public so we don't need to mention it here :)

Reply:
Ah damn, I think it is public.

Can we make it `pub(crate)` as part of this PR? I don't think this should be public API!

Reply:
Ah I remember now, this was added as part of `exec_after_start`!

It is fine as is actually, I guess this has to be part of the public API then. Sorry about the wild-goose chase!